hat/assets/js/apps/Iaso/constants/routes.tsx

```diff
@@ -170,6 +171,13 @@ export const orgUnitChangeRequestPath = {
     element: <ReviewOrgUnitChanges />,
 };
 
+export const orgUnitsChangeRequestConfiguration = {
+    baseUrl: baseUrls.orgUnitsChangeRequestConfiguration,
+    routerUrl: `${baseUrls.orgUnitsChangeRequestConfiguration}/*`,
+    permissions: [Permission.ORG_UNITS_CHANGE_REQUEST_REVIEW],
```

Comment:
this should be ORG_UNITS_CONFIGURATION_CHANGE_REQUESTS no ?

Reply:
also to fit with the rest i would rename it to ORG_UNITS_CHANGE_REQUESTS_CONFIGURATION